dev/com.ibm.ws.jpa.tests.eclipselink.query_jpa_3.2_fat/build.gradle

```diff
@@ -40,6 +41,11 @@ task copyFAT {
     from project(':com.ibm.ws.jpa.tests.eclipselink.query_31_fat.common').file('fat/src/com/ibm/ws/jpa/tests/eclipselink/query/tests/jpa31')
     into new File('fat/src/com/ibm/ws/jpa/tests/eclipselink/query/tests/jpa31')
   }
+  dependsOn ':com.ibm.ws.jpa.tests.eclipselink.query_32_fat.common:build'
```

Comment:
remove this dependsOn.... these files are not needed; already provided by query_31_fat.common